Components/LearningCore/Source/gov/sandia/cognition/learning/algorithm/ensemble/BaggingCategorizerLearner.java

```diff
@@ -119,5 +126,134 @@ protected void addEnsembleMember(
         this.ensemble.add(member, 1.0);
     }
 
+    @Override
+    public int[] getDataInBag()
```

Comment:
This seems dangerous: Returning the raw data will allow the caller the ability to alter our own data directly and voids encapsulation.  Is there a reason not to use ArrayList<Integer> and return Collections.unmodifiableList()?